<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickInfoForm</class>
 <widget class="QWidget" name="PickInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>UUID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="UUID">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="Type">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Texture:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>VsShader:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>PsShader:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>缩放:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="scaleX">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-10000000000000000303786028427003666890752.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000000620008645040778319495168.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="scaleY">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-10000000000000000303786028427003666890752.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="scaleZ">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-10000000000000000303786028427003666890752.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>旋转:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="rotX">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="rotY">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="rotZ">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000000000000303786028427003666890752.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>位移:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="posX">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="posY">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QDoubleSpinBox" name="posZ">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000009190283508143378238084034459715684532224.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Texture"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="VsShader">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="PsShader">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="deleteComponent">
       <property name="text">
        <string>删除该组件</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>GsShader</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="GsShader">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
